### DISCUSSION TITLE
account_activity_times table generation

### DISCUSSION THREAD

**Discussion by a14guguliye** [jeremyevans/rodauth#255] [General]
Hello,
why the account_activity_times table is not generated by rodauth automatically although all others are being generated? Meanwhile, the below code gives me the error. I know it is not an issue but trying to solve it below for weeks now, but can not solve. Can someone please help?
`class New < ActiveRecord::Migration[7.0]
  def change
    create_table(:account_activity_times) do
      foreign_key :id, :account, :primary_key=>true, :type=>:Bignum
      DateTime :last_activity_at, :null=>false
      DateTime :last_login_at, :null=>false
      DateTime :expired_at
    end
  end
end
`

**Comment by jeremyevans** [comment]
Rodauth never creates tables automatically.  You may be using rodauth-rails, please ask on their discussion form: https://github.com/janko/rodauth-rails/discussions/new

**Reply by a14guguliye** [reply]
thanks for ur guidance
